src/scenery.cpp

```diff
@@ -257,8 +257,13 @@ bool SceneryInstance::NeedsWatering() const
 	return this->type->watering_interval > 0 && this->last_watered > this->type->watering_interval;
 }
 
+static const uint32 CURRENT_VERSION = 1;   ///< Currently supported version of %SceneryInstance.
```

Comment:
Just browsing your changes now, this variable needs a suffix it seems.
Will have a closer look either this evening or tomorrow

As you are changing the save format, I expected some data format document change, but that seems lacking.

Reply:
Whoops, that doc is seriously outdated by now. Will give it a full overhaul…